lib/instrumentation.ex

```diff
@@ -15,6 +15,18 @@ defmodule OpentelemetryAbsinthe.Instrumentation do
   require Logger
   require Record
 
+  @telemetry [:opentelemetry_absinthe, :graphql, :handled]
+
+  @type graphql_handled_event_metadata :: %{
+          operation_name: String.t() | nil,
+          operation_type: :query | :mutation,
```

Comment:
Could operation_type be `subscription` too?
Maybe we could refer to the `Conventions` here i.e. 
```
operation_name: Conventions.graphql_operation_name()
operation_type: Conventions.graphql_operation_type()
```

Reply:
This wouldn't actually be correct: `Conventions.graphql_operation_name() = "graphql.operation.name"` and `Conventions.graphql_operation_type() = "graphql.operation.type"`

I am not convinced this could be `:subscription` too, there are specific metrics related to subscriptions in absinthe.

Reply:
Haha you're right, i'm an idiot!